packages/core/integration-tests/test/transpilation.js

```diff
@@ -179,6 +180,26 @@ describe('transpilation', function() {
     assert(!file.includes('es.array.concat'));
   });
 
+  it('should resolve @swc/helpers and regenerator-runtime relative to parcel', async function() {
+    let dir = path.join(
+      '/tmp/' +
+        Math.random()
+          .toString(36)
+          .slice(2),
+    );
+    await outputFS.mkdirp(dir);
+    ncp(path.join(__dirname, '/integration/swc-helpers'), dir);
```

Comment:
What is this tmp directory for?

Reply:
Just so it's outside the monorepo. Otherwise it'll still resolve the helpers